pkg/services/dashboards/database/database.go

```diff
@@ -640,6 +640,26 @@ func (d *dashboardStore) GetDashboardsByPluginID(ctx context.Context, query *das
 	}
 	return dashboards, nil
 }
+func (d *dashboardStore) GetSoftDeletedDashboard(ctx context.Context, query *dashboards.GetDashboardQuery) (*dashboards.Dashboard, error) {
+	var queryResult *dashboards.Dashboard
+	err := d.store.WithDbSession(ctx, func(sess *db.Session) error {
+
+		dashboard := dashboards.Dashboard{OrgID: query.OrgID, UID: query.UID}
+		mustCols := []string{}
```

Comment:
What is the purpose here of mustCols?

Reply:
It's used to append the query search below. I referenced the original [GetDashboard](https://github.com/grafana/grafana/blob/main/pkg/services/dashboards/database/database.go#L812) method. I could write it out all in one line as well

Reply:
It shouldn't be necessary unless you specify some columns, in this case is empty so you can remove it before the `.Get`